cfme/utils/appliance/__init__.py

```diff
@@ -1819,7 +1819,8 @@ def is_embedded_ansible_role_enabled(self):
 
     @property
     def is_embedded_ansible_running(self):
-        return self.is_embedded_ansible_role_enabled and self.supervisord.running
+        supervisord = self.supervisord.running if self.version < '5.11' else True
+        return self.is_embedded_ansible_role_enabled and supervisord
```

Comment:
```suggestion
        return self.is_embedded_ansible_role_enabled and (self.version < '5.11' == self.supervisord.running)
```

Reply:
@jaryn there is no `supervisord` service to check in terms of `appliance` `version` greater than or equal to 5.11 would fail and return `False`.

Reply:
@sbulage You are right. But that doesn't mean my suggestion is wrong.

Reply:
@jaryn I am not saying it is wrong :smile:. I will update it.